### ISSUE TITLE
Drag and drop not working

### ISSUE THREAD

**Issue by luni3359** [flathub/com.github.taiko2k.avvie#2]
I'm on KDE neon User Edition 5.18 x86_64

``flatpak override --show --user com.github.taiko2k.avvie``
```
[Context]
filesystems=host;home
```

``flatpak override --show com.github.taiko2k.avvie ``
```
[Context]
filesystems=host;
```

**Comment by Taiko2k** [comment]
Drag and drop is expected to be working, and does work in KDE on my machine.

Can you drag and drop from anywhere, such as the folders with permission by default like the Pictures folder?

Do you encounter this issue with other GTK apps installed by Flatpak?

**Comment by luni3359** [comment]
Now that you point it out, yes, it seems like I'm capable to, but if I drag from the DE desktop it doesn't work (not from Nautilus).

**Comment by Taiko2k** [comment]
Yes I can replicate this behavior. I can't drag and drop files from the Plasma desktop either. It seems to affect most GTK applications (with the exception I found being Nautilus).

They just show a red disabled icon type cursor when trying to drop over the window. I can confirm this isn't a Flatpak related issue as its still occurs for apps outside the sandbox.

My guess is its a GTK related bug.

**Comment by luni3359** [comment]
Have you reported it?

**Comment by Taiko2k** [comment]
OK, I have opened a report here https://gitlab.gnome.org/GNOME/gtk/-/issues/2567
